code/modules/mob/delays.dm

```diff
@@ -34,7 +34,7 @@
 // Constructor args are currently all the same, but placed here for ease of tuning.
 /client // Yep, clients are snowflakes.
 	// Walking speed is 7, as is grab speed.
-	var/datum/delay_controller/move_delayer    = new (1,ARBITRARILY_LARGE_NUMBER) // /mob/delayNextMove()
+	var/datum/delay_controller/move_delayer    = new (0.001,ARBITRARILY_LARGE_NUMBER) // /mob/delayNextMove()
```

Comment:
In testing, I've learned that a delay of 0.4 doesn't allow mobs to move quite as fast as I'd like. How about 0.3?